x-pack/plugins/index_lifecycle_management/public/application/services/api.ts

```diff
@@ -15,8 +14,13 @@ import {
   UIM_INDEX_RETRY_STEP,
 } from '../constants';
 
+import { trackUiMetric } from './ui_metric';
 import { sendGet, sendPost, sendDelete, useRequest } from './http';
 
+interface GenericObject {
```

Comment:
I defined these locally because they're not part of the ILM domain, so I don't think they belong in a common `types` file.